### PR TITLE
LNX-188 Add Drop Action

### PR DESCRIPTION
**Pull request by EasyJB**

### CODE REVIEW

**Review by Krystian030 — APPROVED** [review]

**Review by Krystian030 — APPROVED** [review]

File: src/runtime.py [hunk 1]
```diff
@@ -38,7 +39,6 @@ def send(action: Action):
         logger.debug(f"Scene has been successfully received, starting to deserialize scene")
         scene = Scene.deserialize(scene_serialized)
         logger.debug(f"Scene has been successfully deserialized")
-
```
Comment (filip-szweda) [inline src/runtime.py]: Please rollback this change. There must be an empty line between to functions

File: src/app.py [hunk 2]
```diff
@@ -18,4 +18,4 @@ async def lifespan(app: FastAPI):
 app = FastAPI(lifespan=lifespan)
 app.add_middleware(GZipMiddleware, minimum_size=100)
 app.include_router(scene_router.router)
-app.include_router(server_router.router)
```
Comment (filip-szweda) [inline src/app.py]: Please rollback this change. Without it, diff will think that adding anything to the end of this file changes `app.include_router(server_router.router)`

**Review by filip-szweda — CHANGES_REQUESTED** [review]

File: src/runtime.py [hunk 3]
```diff
@@ -93,7 +94,8 @@ def goto(target_position: Vector):
         'calculate_distance_to': lambda id: calculate_distance_to(scene, id, object_id),
         'random_direction': lambda: random_direction(scene, object_id),
         'print': lambda text: send(Print(str(text), scene.get_object_by_id(object_id).owner)),
-        'take': lambda vector: send(Take(object_id, vector))
+        'take': lambda vector: send(Take(object_id, vector)),
+        'drop': lambda target_position: send(Drop(object_id, target_position)),
```
Comment (filip-szweda) [inline src/runtime.py:98]: No `,` needed on the end of this entry, because it is the last one

**Review by filip-szweda — APPROVED** [review]

File: src/runtime.py (hunk 3 above)
Comment (Krystian030) [reply src/runtime.py:98]: It useful when we compare the difference between changes.
https://stackoverflow.com/questions/11597901/why-are-trailing-commas-allowed-in-a-list

**Pull request merged**